### PR TITLE
CTCP-187 Initial scaffold to demo GG redirection

### PR DESCRIPTION
**Pull request by crystalmark**

### CODE REVIEW

**Review by RobertLaverick — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -1,8 +1,181 @@
...
+This version of the code only demonstrates how to redirect to the Government Gateway sign-in pages.  
 
-### License
+We are working on an example page to submit a new departure declaration message.
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
\ No newline at end of file
+## The Example Application
+
+This code runs Sping Boot with an Apache Wicket UI framework.  It is configured to run using the HMRC Sandbox environment.
```
Comment (liandra-giacomine) [inline README.md:11]: Sping Boot -> Spring Boot?

Comment (crystalmark) [reply README.md:11]: Thanks Liandra.  Good spot!

**Review by liandra-giacomine — APPROVED** [review]
looks good

**Pull request merged**